config/initializers/5_telemetry.rb

```diff
@@ -1,4 +1,3 @@
-require 'foreman/telemetry'
```

Comment:
I am not sure what you mean, I tested my patch, it works and it passes tests.

Telemetry must be initialized as one of the first things, that's why the initializer is named 5_ since we use telemetry calls during app initialization.

Can we just merge this which fixes app so it can be actually started when telemetry is turned on? I have no intentions on working on auto loading, it has never worked and I have my doubts it will.

Reply:
Well we need to switch to `Zeitwerk` and that prevents us from using autoloading at this stage. So it works now, but it won't once we enable Zeitwerk and that was what the original patch was aiming to prepare us for.
I'll play around it a bit and see what we can do :)